src/components-examples/cdk/drag-drop/cdk-drag-drop-table/cdk-drag-drop-table-example.css

```diff
@@ -0,0 +1,28 @@
+table {
+  width: 100%;
+}
+
+.example-drag-cursor {
+  margin-right: 16px;
+  cursor: move;
+}
+
+.cdk-drag-preview {
```

Comment:
Should we add a background color to this one? Currently it's transparent.

Reply:
Yeah, I'll add it.